code-env/python/spec/requirements.txt

```
@@ -0,0 +1,20 @@
+symspellpy==6.5.2
+spacy>=2.2.2
```

Use == for versions, or >=X.x,<Y.y
```suggestion
spacy==2.3.0
```